### DISCUSSION TITLE
container/CRI

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#246] [General]
# container/CRI

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/container/CRI.html

**Comment by useforgithub** [comment]
Kata Containers 本质上是通过虚拟化技术模拟出一台“微型虚拟机”.
是不是可以理解为在传统虚拟化的基础上，运行更精简的Linux？
最终回到了虚拟机，那Docker还需要存在吗？

**Reply by isno** [reply]
启动一个简单的容器进程（nginx），没必要带着一个 Linux 内核。
Kata 是为了解决容器共享内核问题，它和 Docker 不冲突。
